docs/javascript/v2/how--to-guides/record-and-live-stream/hls/hls-timed-metadata.mdx

```diff
@@ -17,7 +17,7 @@ Currently there is a limit to send at max 3 metadata at time.
 
 ```javascript
 export interface HLSTimedMetadata {
-	payload: string; // limited to 100 chars, no " (double quotes) allowed
+	payload: string; // limited to 100 chars
 	duration: number;
 }
 
```

Comment:
`The payload is sent in the EXT-X-DATERANGE tag as value for the custom attribure X-100MSLIVE-PAYLOAD. It is base64 encoded string of` - this sentence is incomplete on the page

Reply:
```start_date and end_date will be in UTC format.```

UTC is just a timezone. the format is ISO-8601

Reply:
how is the user going to use the information around `EXT-X-DATERANGE`?

Reply:
They will need to convert base64 data to stringify json.